content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,241 @@
+| Intended use | Pods | People |
+
+{{< /table >}}
+
+### Default service accounts {#default-service-accounts}
+
+When you create a cluster, Kubernetes automatically creates a ServiceAccount
+object named `default` for every namespace in your cluster. The `default`
+service accounts in each namespace get no permissions by default other than the
+[default API discovery permissions](/docs/reference/access-authn-authz/rbac/#service-account-permissions)
+that Kubernetes grants to all authenticated accounts. If you delete the
+`default` ServiceAccount object in a namespace, the
+{{<glossary_tooltip text="control plane" term_id="control-plane">}}
+replaces it with a new one.
+
+If you deploy a Pod in a namespace, and you don't manually assign a
+ServiceAccount object to the Pod, Kubernetes assigns the `default` service
+account for that namespace to the Pod and provides the credentials for that
+ServiceAccount to the Pod. To prevent Kubernetes from automatically providing
+credentials for the `default` ServiceAccount, set the
+`automountServiceAccountToken` field in your Pod specification to `false`.
+
+## Use cases for Kubernetes service accounts {#use-cases}
+
+As a general guideline, you can use service accounts to provide identities in
+the following scenarios:
+
+* Your Pods need to communicate with the Kubernetes API server, for example in
+  situations such as the following:
+  *  Providing read-only access to sensitive information stored in Secrets.
+  *  Granting [cross-namespace access](#cross-namespace), such as allowing a
+     Pod in namespace `example` to read, list, and watch for Lease objects in
+     the `kube-node-lease` namespace.
+* Your Pods need to communicate with an external service. For example, a
+  workload Pod requires an identity for a commercially available cloud API,
+  and the commercial provider allows configuring a suitable trust relationship.
+* [Authenticating to a private image registry using an `imagePullSecret`](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
+* An external service needs to communicate with the Kubernetes API server. For
+  example, authenticating to the cluster as part of a CI/CD pipeline.
+* You use third-party security software in your cluster that relies on the
+  ServiceAccount identity of different Pods to group those Pods into different
+  contexts.
+
+
+## How to use service accounts {#how-to-use}
+
+To use a Kubernetes service account, you do the following:
+
+1. Create a ServiceAccount object using a Kubernetes
+   client like `kubectl` or a manifest that defines the object.
+1. Grant permissions to the ServiceAccount object using an authorization
+   mechanism such as
+   [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+1. Assign the ServiceAccount object to Pods during Pod creation.
```

Comment:
3b. retrieve the ServiceAccount token and use it from an external service

Reply:
DOne